webpack.config.js

```diff
@@ -156,6 +168,7 @@ const webpackConfig = {
 				transform: content => content,
 			} ) )
 		),
+
```

Comment:
There is no need for this empty line, right? :upside_down_face: 
```suggestion
```

Reply:
Thanks, fixed in https://github.com/woocommerce/wc-admin/pull/1500/commits/45badff93d44a9920192796be5875bab131c0a93